### PR TITLE
Added accomodation conference text

### PR DESCRIPTION
**Pull request by AndrewBestbier**

### CODE REVIEW

File: shared/components/NextConferenceEvent/index.scss [hunk 1]
```diff
@@ -25,7 +25,29 @@
...
+      font-size: 18px;
+    }
+    *:last-child {
+      margin-bottom: 0;
+    }
+  }
+  &__link {
+    font-weight: 700;
+    text-decoration: underline;
+    color: #4a4a4a;
```
Comment (asavin) [inline shared/components/NextConferenceEvent/index.scss]: hardcoded color

File: shared/components/NextConferenceEvent/index.scss [hunk 2]
```diff
@@ -25,7 +25,29 @@
       }
     }
   }
-
+  &__accomodation{
+    > h2 {
+      margin: 0;
+      border-bottom: 2px dashed #4a4a4a;
```
Comment (asavin) [inline shared/components/NextConferenceEvent/index.scss]: hardcoded color

Comment (zoeabryant) [inline shared/components/NextConferenceEvent/index.scss]: use $dash-border in the variables file

File: shared/components/NextConferenceEvent/index.scss (hunk 1 above)
Comment (zoeabryant) [inline shared/components/NextConferenceEvent/index.scss]: use $grey variable

File: shared/components/NextConferenceEvent/index.scss (hunk 2 above)
Comment (AndrewBestbier) [inline shared/components/NextConferenceEvent/index.scss]: +1

File: shared/components/NextConferenceEvent/index.scss (hunk 1 above)
Comment (AndrewBestbier) [inline shared/components/NextConferenceEvent/index.scss]: +1

File: shared/components/NextConferenceEvent/index.js [hunk 3]
```diff
@@ -45,6 +45,24 @@ const NextConferenceEvent = ({ calendarURL }) => (
           </ExternalLink>
         </div>
       </article>
+      <div className="NextConferenceEvent__accomodation">
+        <h2>Plan your visit</h2>
+        <p>
+          We have negotiated a limited number of rooms at
+          the <ExternalLink
+            href={calendarURL}
```
Comment (asavin) [inline shared/components/NextConferenceEvent/index.js]: this is wrong

**Pull request merged**